solr/core/src/java/org/apache/solr/util/plugin/AbstractPluginLoader.java

```diff
@@ -86,7 +86,7 @@ public AbstractPluginLoader(String type, Class<T> pluginClassType)
    * @param node - the XML node defining this plugin
    */
   @SuppressWarnings("unchecked")
-  protected T create( SolrResourceLoader loader, String name, String className, Node node ) throws Exception
+  protected T create(SolrClassLoader loader, String name, String className, Node node ) throws Exception
```

Comment:
BTW I do like these sorts of changes where you've found SRL used where the caller really just needs a class loader.